Automattic/jetpack
# Jetpack: Check whether the Manager::is_site_connection method exists

#### Changes proposed in this Pull Request:
* If an older version of the Manager class is loaded when Jetpack activates, a fatal error can occur when `Manager::is_site_connection()` is called. Check whether it exists before calling it.

#### Jetpack product discussion
* p91TBi-66k-p2

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
##### Cause the fatal error
1. Install and activate the WooCommerce and WooCommerce Payments plugins.
2. Connect WooCommerce Payments to WPCOM (you don't need to set up a Stripe account.)
3. Install the latest stable version of Jetpack but don't activate it.
4. Activate Jetpack via cli: `wp plugin activate jetpack`.
5. Activation should fail with a fatal error.

##### Test the fix
1. Clear the autoloader plugins transient to make sure that Jetpack isn't in the transient: `wp transient delete jetpack_autoloader_plugin_paths.`
2. Make sure that WooCommerce Payments is connected. You can connect using the `Payments` page or check for the connection tokens using `wp option get jetpack_private_options`.
3. Install this branch of Jetpack but don't activate it.
4. Activate Jetpack via cli: `wp plugin activate jetpack`.
5. Activation should be successful and no error should be generated.
